queries/asciidoc/injections.scm

```diff
@@ -0,0 +1,44 @@
+((paragraph) @injection.content
+  (#set! injection.include-children)
+  (#set! injection.language "asciidoc_inline"))
+
+((line) @injection.content
+  (#set! injection.include-children)
+  (#set! injection.language "asciidoc_inline"))
+
+((section_block
+  (element_attr
+    (element_attr_marker)
+    (attr_value) @injection.language
+    (element_attr_marker))
+  (raw_block
+    (raw_block_marker)
+    (raw_block_body) @injection.content
+    (raw_block_marker)))
+  (#gsub! @injection.language "%s*,%s*(%w+)" "%1"))
+
+((section_block
+  (element_attr
+    (element_attr_marker)
+    (attr_value) @injection.language
+    (element_attr_marker))
+  (raw_block
+    (raw_block_marker)
+    (raw_block_body) @injection.content
+    (raw_block_marker)))
+  (#gsub! @injection.language "%s*source%s*,%s*(%w+)%s*" "%1")
+  (#vim-match? @injection.language "\\s*source\\s*,\\s*.*\\s*"))
```

Comment:
It would be better to handle this by special-casing it in the parser if possible.
```clojure
(section_block
  (element_attr
    (element_attr_marker)
    (attr_value
      (source_language) @injection.language)
    (element_attr_marker))
  (raw_block
    (raw_block_marker)
    (raw_block_body) @injection.content
    (raw_block_marker)))
```

Reply:
If that's not possible, then I think the (singular) query should look like this:
```clojure
((section_block
  (element_attr
    (element_attr_marker)
    (attr_value) @injection.language
    (element_attr_marker))
  (raw_block
    (raw_block_marker)
    (raw_block_body) @injection.content
    (raw_block_marker)))
  (#gsub! @injection.language "^[^,]*,%s*(%w+)" "%1")
  (#lua-match? @injection.language "^[^,]*,%s*(%w+)"))
```

Reply:
Similar to the following. `attr_value` cannot be any value. Otherwise it is not a code block.

Reply:
[But it can.](https://docs.asciidoctor.org/asciidoc/latest/verbatim/highlight-lines/)